### DISCUSSION TITLE
How do I import questions with answers and explanation to wp-adv-quiz from an excel file?

### DISCUSSION THREAD

**Discussion by ivanli0** [markusbegerow/Wp-Adv-Quiz#53] [General]
Love the app. I have a bunch of questions from an excel file which I wish to import into into the plugin. Is there anywhere I can do this without writing them individually. It only accepts *.waq, *.xml files which I don't quite understand. Thank you, would be grateful to hear from you. I am hoping to import MCQ questions with 4-5 answer choices and explanations for the correct answers.

<img width="781" alt="image" src="https://github.com/markusbegerow/wp-adv-quiz/assets/122064508/f4dec65d-26c9-41fd-88dd-80fa0914de1f">

**Comment by markusbegerow** [comment]
@ivanli0 Please make a proper propsal to implement this

**Reply by ivanli0** [reply]
Thank you for your email. How do I do this?

**Reply by markusbegerow** [reply]
Goto https://github.com/markusbegerow/wp-adv-quiz/issues

make the prop.
Should I integrate your in prop in our poll?
